core/src/runtime/opaque.rs

```diff
@@ -50,6 +58,7 @@ impl<'js> Opaque<'js> {
         Opaque {
             panic: Cell::new(None),
             interrupt_handler: UnsafeCell::new(None),
+            class_ids: TinyVec::from([0; 1024]),
```

Comment:
This TinyVec here seems kinda unnecessary. Since Opaque is allocated on the heap the TinyVec is also, your essentially pre-allocating `1024 * sizeof(JSClassID)` bytes for a buffer. If this is required for performance it would probably also be fine to just use a normal vec and call `Vec::with_capacity(1024)` the single omitted allocation doesn't really justify the inclusion of a new dependency to me.

Reply:
Yeah not worth it. This will all go away once the new class system is in place. I will rebase after that is merged.